website/docs/cloud-docs/api-docs/private-registry/modules.mdx

```diff
@@ -546,7 +548,8 @@ Properties without a default value are required.
   "data": {
     "type": "registry-module-versions",
     "attributes": {
-      "version": "1.2.3"
+      "version": "1.2.3",
+      "commit_sha": "abcdef12345"
```

Comment:
```suggestion
      "commit-sha": "abcdef12345"
```

Reply:
my apologies for missing this, ty brian!